.travis.yml

```diff
@@ -1,13 +1,14 @@
 language: scala
-script:
-  - ./bin/runci.sh $CI_TEST
-jdk:
-  - oraclejdk8
+scala: 2.12.3
```

Comment:
What does this line do?

Reply:
Purely curious. I've seen it used elsewhere but never understood what it does.

Reply:
It’s useful for projects that cross compiles for multiple scala versions since if contributes to the build matrix. In the case of metadata it doesn’t make much difference, however, I was annoyed with seeing 2.10 reported as the scala version